agent/api/containerstatus.go

```diff
@@ -49,9 +49,28 @@ const (
 	ContainerUnhealthy
 )
 
+const (
+	// ContainerNotTransition means the container isn't in the middle of a transition
+	ContainerNotTransition ContainerTransitioningStatus = iota
+	// ContainerPulling means the container is in the process of pulling
+	ContainerPulling
```

Comment:
Can you make sure that `ContainerTransition` is a prefix for all of these? Example: `ContainerTransitionPulling`